MetaMask/metamask-extension
# Upgrade storybook from 7.4.1 => 7.5.3

## **Description**

This PR upgrades storybook from `7.4.6` => `7.5.3`. The recordings below check the following
- MDX documentation
- Stories are rendered correctly and can be interacted with
- Stories display correctly in the MDX documentation in the Docs tab
- Args table showing props render correctly in the MDX documentation  
- Controls work
- Other storybook plugins including Actions, Accessibility, HTML work

## **Related issues**

Fixes: #21872

## **Manual testing steps**

1. Pull this branch or go to the latest build of storybook in this PR
2. Check MDX documentation, stories, controls and plugins

## **Screenshots/Recordings**

### **Before**


https://github.com/MetaMask/metamask-extension/assets/8112138/6a73a64a-ecce-47e9-8a88-477223282166

### **After**


https://github.com/MetaMask/metamask-extension/assets/8112138/994e0a72-5ac1-4ef0-b284-2315a64d1dbe


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
